packages/front/src/components/ClaimCreate.jsx

```diff
@@ -5,26 +5,26 @@ import { Editor, EditorState } from 'draft-js'
 import { stateToHTML } from 'draft-js-export-html'
 import 'draft-js/dist/Draft.css';
 
-class SpeechCreate extends Component {
+class ClaimCreate extends Component {
     constructor(props) {
         super(props);
 
         this.state = {
             editorState: EditorState.createEmpty(),
         }
 
-        this.saveSpeech = this.saveSpeech.bind(this);
+        this.saveClaim = this.saveClaim.bind(this);
         this.onChange = (editorState) => this.setState({editorState});
       }
 
-    saveSpeech(e) {
+    saveClaim(e) {
         e.preventDefault();
 
         const html = stateToHTML(this.state.editorState.getCurrentContent());
         const title = this.state.title;
         const personality = this.props.match.params.personalityId;
         console.log({ title, html, personality });
-        axios.post('http://localhost:3000/speech', { title, html, personality })
+        axios.post('http://localhost:3000/claim', { title, html, personality })
```

Comment:
We can remove this and others requester for folder where there happening requesters by methods. What do you thing?

Reply:
Do you mean using a base URL?

Reply:
Yes.